backend/config/settings.py

```
@@ -34,8 +32,16 @@ def safe_context_copy(self):
 BaseContext.__copy__ = safe_context_copy
 
-BASE_DIR = Path(__file__).resolve().parent.parent
-PLUGINS_DIR = BASE_DIR / "plugins"
+STRIPE_PUBLISHABLE_KEY = os.getenv('STRIPE_PUBLISHABLE_KEY')
+STRIPE_SECRET_KEY = os.getenv('STRIPE_SECRET_KEY')
+STRIPE_WEBHOOK_SECRET = os.getenv('STRIPE_WEBHOOK_SECRET')
+
+stripe.api_key = STRIPE_SECRET_KEY
+
+
+def load_dotenv(dotenv_path: Path) -> None:
```

Ruff (0.15.21)

[error] Undefined name `Path`

(F821)